.github/pull_request_template.md

```diff
@@ -1,61 +1,36 @@
 <!--
-- - -  I N S T R U C T I O N S  --  P L E A S E  R E A D  - - -
-Please remove boiler plate instructions when filling out this template.
-Please remove unwanted/unrelated/irrelevant information such as comments provided as a reference in this template.
-Use proper formatting to separate code snippets from text description e.g. use ```code block``` or `in-line code`.
-Please copy any output files into a Github gist (for e.g.) and link to the gist, rather than relying on paths on platforms that might change or disappear.
-
-Before opening a PR, please note these guidelines:
-
-- Each PR should only address ONE topic and have an associated GitHub issue
-- No hardcoded or paths to personal directories should be present
-- No temporary or backup files should be committed
-- Any code that was disabled by being commented out should be removed
-- Please provide a on-line, but descriptive title to the PR e.g. Adds XYZ functionality or Fixes ABC bugs, etc.  DO NOT use branch names as titles.
--->
 
-**Description**
+INSTRUCTIONS
 
-<!-- Please include relevant motivation and context. -->
-<!-- Please include a summary of the change and which issue is fixed. -->
-<!-- List any dependencies that are required for this change. -->
+- Please do not commit temporary, backup, or binary files.
+- Please remove commented-out code.
+- Please ensure code, config files, etc., contain no hardcoded paths.
+- Please format code snippets in PR description/comments with ```code block``` or `inline code`.
+- Please consider adding your own review comments to guide other reviewers.
+
+-->
 
-<!-- Please provide reference to the issue this pull request is addressing. -->
-<!-- For e.g. Fixes #IssueNumber -->
+**Description**
 
-**Type of change**
+<!-- A summary of the change, including relevant motivation, context, useful links, etc. -->
 
-Please delete options that are not relevant.
+**Type**
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
+Select one or more:
 
-**How Has This Been Tested?**
+- [ ] Bug fix (corrects a known issue)
+- [ ] Code maintenance (refactoring, etc.)
+- [ ] New feature (adds functionality)
 
-<!-- Please describe the tests that you ran to verify your changes. -->
-<!-- Provide instructions so we can reproduce. -->
-<!-- Please also list any relevant details for your test configuration -->
+**Impact**
 
-<!-- Use the following as a guide to list your tests and delete options that are not relevant. Expand as necessary. -->
-<!--
-- [ ] pytests in GitHub actions.
-- [ ] Tests on XYZ OS/HPC/CSP
```

Comment:
This was somewhat left over from early evaluations of platforms, which we may need to re-include if we have to do any manual platform tests outside of Jenkins

Reply:
This is a good point, though I can't immediately see why there would be anything platform-specific in UW that would require testing outside of Actions. Even the kinds of integration tests I think we'd eventually want to to run at least on `develop` -> `main` merges to create releases should probably be testable in a generic Linux container like Actions provides. Integration tests for downstream apps are where I'd expect to see platform concerns. Please point out anything I might be missing...

Reply:
True! That would certainly be for integration tests. I think some will have to be run on specific tier 1 machines rather than just a generic container. For what its worth, SRW PRs ask which specific machines were tested on (which was simplified here), while the main UFS specifically asks "I have run the full RT suite on either Hera/Cheyenne AND have attached the log to this PR below this line"